modules/common/src/main/java/org/opencastproject/security/util/SecurityUtil.java

```diff
@@ -164,7 +164,11 @@ public static String getSystemUserName(ComponentContext cc) {
 
   /** Extract hostname and port number from a URL. */
   public static Tuple<String, Integer> hostAndPort(URL url) {
-    return tuple(StringUtils.strip(url.getHost(), "/"), url.getPort());
+    int port = url.getPort();
+    if (port < 0) {
```

Comment:
Previously, `port=-1` meant any port. Does this now mean something else?

Reply:
When a URL is parsed, `-1` means that no port is explicitly specified in the URL. We can also check for `-1`, but every port number below 0 is invalid.

Reply:
Can this be resolved?

Reply:
?

Reply:
Don't wait for me. If this is fine, it's fine. It was just something I noticed. I didn't do a proper review of this.